src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEAssemblySymbol.cs

```diff
@@ -90,6 +91,15 @@ public override AssemblyIdentity Identity
             }
         }
 
+        public override Version AssemblyVersionInSource
+        {
+            get
+            {
+                // TODO: https://github.com/dotnet/roslyn/issues/9000
+                return _assembly.Identity.Version;
```

Comment:
If we are not supporting AssemblyVersionInSource for PEAssemblySymbols, I think we should return null here. There will be less chances for a break when we start supporting it.